log/README.md

```diff
@@ -1,7 +1,7 @@
 # Basic usage
 
 ```ts
-import * as log from "https://deno.land/x/std/logging/index.ts";
+import * as log from "https://deno.land/x/std/log/mod.ts";
```

Comment:
Is `std` keep in the url necessary?

Reply:
Maybe not, so [database.json](https://github.com/denoland/registry/blob/master/src/database.json) needs to be changed. right?

Reply:
https://github.com/denoland/registry/pull/29
@zhmushan Is this OK?